src/azure-cli-core/azure/cli/core/profiles/shared.py

```diff
@@ -78,7 +133,15 @@ def get_client_class(resource_type):
     :raises: ImportError, AttributeError
     :returns:  class -- the Client class.
     """
-    return getattr(import_module(resource_type.sdk_module), resource_type.client_name)
+    cp = resource_type.client_path
+    try:
+        mod_to_import, attr_path = cp.split('#')
+        op = import_module(mod_to_import)
+        for part in attr_path.split('.'):
+            op = getattr(op, part)
+        return op
+    except (ValueError, AttributeError):
+        return None
```

Comment:
It couldn't get the client class.
Caller shouldn't need to handle this as it shouldn't happen.
**If** it does happen, there's not much the caller can do as they needed the client for some purpose.

@yugangw-msft As an alternative, I can remove the try/except and just let the exception be uncaught. Thoughts?